### DISCUSSION TITLE
Searching the journal does not seem to work (anymore?)

### DISCUSSION THREAD

**Discussion by ixici** [matthiasn/lotti#1038] [Improvements]
<img width="618" alt="Screenshot 2022-06-09 at 21 30 33" src="https://user-images.githubusercontent.com/55126809/172919120-3ad31a3f-9cf0-4077-935f-c17c9be0f48b.png">

**Comment by matthiasn** [comment]
Currently search is only implemented for tags, with full-text search still on my backlog. Will focus on journaling functionality again when sync assistant works properly, plus some minor improvements around dashboards.

**Comment by matthiasn** [comment]
Converting this to a discussion around full-text search. Please upvote for prioritizing full-text search.

**Reply by matthiasn** [reply]
Full-text search is working again. Please run the maintenance task for recreation of the index to have existing entries included, otherwise only new entries will be indexed.
